boards/adi/adi_eval_adin1110ebz/doc/index.rst

```diff
@@ -82,7 +82,7 @@ following hardware features:
 
 The default configuration can be found in the defconfig file:
 
-	``boards/arm/adi_eval_adin1110ebz/adi_eval_adin1110ebz_defconfig``
+	``boards/adi/adi_eval_adin1110ebz/adi_eval_adin1110ebz_defconfig``
```

Comment:
:zephyr_file:`boards/adi/adi_eval_adin1110ebz/adi_eval_adin1110ebz_defconfig`. Could be done in a single commit.